src/test/java/org/apache/maven/shared/dependency/analyzer/DefaultProjectDependencyAnalyzerTest.java

```diff
@@ -427,6 +427,32 @@ public void testTypeUseAnnotationDependencyOnLocalVariable()
         assertEquals( expectedAnalysis, actualAnalysis );
     }
 
+    public void testUnnamedPackageClassReference()
+        throws TestToolsException, ProjectDependencyAnalyzerException
+    {
+        if ( !SystemUtils.isJavaVersionAtLeast( JavaVersion.JAVA_1_8 ) )
```

Comment:
This is a use case for Assume.

Reply:
👍 updated, let me know if you think I should make the same change to other tests in this class